deploy/README.md

```diff
@@ -0,0 +1,225 @@
+# Deploying Bubbles for Development
+
+Given Bubbles is not part of the main ceph distribution, we need to take a few
```

Comment:
```suggestion
Given Bubbles is not part of the main Ceph distribution, we need to take a few
```

Reply:
> > there are quite a few type changes in this PR. Should we start validating these (#8)?
> 
> yeah, on a different PR though ;)

agree, #8 is still wip, there's still plenty that needs to be done to get pytest to succeed